Update README and other documentation (#711)
nsidc/qgreenland: doc/user/tutorials/video-series-overview.md

# Beginner Video Series

## Overview

In this video series (~1 hour total), you will become familiar with QGIS,
download and browse the QGreenland Core data package, and explore the basic

> geospatial data anlysis tools that make QGreenland a powerful tool for research,

Reviewer: ```suggestion
geospatial data anlysis tools that make QGIS and QGreenland powerful tools for research,
```

education, and local decision making. The video tutorial series contains 7 short
video sessions.

## Goals

* Learn what data is included in the QGreenland Core Package
* Analyze Greenland data using geoprocessing tools
* Find analysis tools and data relevant to you or your classes

## Prerequisites

In order to get the most out of this series, we recommend the following:
* Follow the [Get started with QGreenland Core](/user/tutorials/get-started.md)
  tutorial for directions on installing QGIS and downloading QGreenland
* When watching the videos, click on the settings icon to adjust the subtitles
  (in English or other languages) and adjust the playback speed
* Take time to engage in the practice tasks and take notes

## Videos

The entire QGreenland Beginner's Tutorial Series playlist can be found
[here](https://www.youtube.com/playlist?list=PLSRiyMridUCwyu-vqpAFtm8bVERgTvs7q). A
brief description of what is covered in each video is given below:

### [Session 1: What is QGreenland](https://www.youtube.com/watch?v=gD0vkP5JUmA&list=PLSRiyMridUCwyu-vqpAFtm8bVERgTvs7q&index=1)

If you are not familiar with geographic information systems (GIS) and geospatial

> data, this video will introduce you to GIS and what it can be used for.

Reviewer: :100:
